jinahub/rankers/LightGBMRanker/tests/integration/test_lightgbm.py

```diff
@@ -0,0 +1,59 @@
+__copyright__ = "Copyright (c) 2020-2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+
+import pytest
+from jina import Flow
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+@pytest.fixture
+def model_path(tmpdir):
```

Comment:
Use `tmp_path`, `tmpdir` is based on a deprecated library + you can use `pathlib` methods with `tmp_path`